spring-cloud/spring-cloud-dataflow-ui
# gh-526 Update to Angular 5 (Incl CLI 1.5)

* Update versions to Angular `5.2.3` and Angular CLI `1.6.7`
* Compare and update all relevant files from a freshly created angular CLI project
* Fix issues around Angular2 Busy and RxJs
  - Use the latest GitHub version of Angular2 Busy
  - Polish RxJs imports
* Remove StompJS for (Not used and has issues with Angular 5)
* Update `ngx-bootstrap` to `2.0.0-beta.8` to fix build error
* Update `ngx-clipboard` from `8.1.3` to `9.0.0`
* Update `compodoc` from `1.0.3` to `1.0.4`
* Update `d3` from `4.11.0` to `4.12.0`
* Switch from `template` to `ng-template` to get tooltips working again
* Switch from PhantomJS to headless Chrome for test executions
* Ensure Maven build passes

resolves https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/526
